deps/Makefile

```diff
@@ -485,12 +493,24 @@ LLVM_MFLAGS += OPTIONAL_PARALLEL_DIRS=clang
 else
 # block default building of Clang
 LLVM_MFLAGS += OPTIONAL_PARALLEL_DIRS=
+ifeq ($(LLVM_VER),svn)	# changed since LLVM 3.9
```

Comment:
If you want to be future-proof, you could also check for `LLVM_VER >= 3.9` (in addition to `svn`).